core/templates/pages/contributor-dashboard-page/translation-language-selector/review-translation-language-selector.component.spec.ts

```diff
@@ -0,0 +1,305 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the translation language selector component.
+ */
+
+import { ComponentFixture, fakeAsync, flush, TestBed, waitForAsync } from '@angular/core/testing';
+
+import { ReviewTranslationLanguageSelectorComponent } from
+  // eslint-disable-next-line max-len
+  'pages/contributor-dashboard-page/translation-language-selector/review-translation-language-selector.component';
+import { ContributionOpportunitiesBackendApiService } from
+  // eslint-disable-next-line max-len
+  'pages/contributor-dashboard-page/services/contribution-opportunities-backend-api.service';
+import { TranslationLanguageService } from 'pages/exploration-editor-page/translation-tab/services/translation-language.service';
+import { ElementRef, EventEmitter } from '@angular/core';
+import { AppConstants } from 'app.constants';
+import { FormsModule } from '@angular/forms';
+
+describe('Review Translation language selector', () => {
+  let component: ReviewTranslationLanguageSelectorComponent;
+  let fixture: ComponentFixture<ReviewTranslationLanguageSelectorComponent>;
+  let translationLanguageService: TranslationLanguageService;
+  let activeLanguageChangedEmitter = new EventEmitter();
```

Comment:
nit: Prefer `const` for any variable that is not being reassigned. Here and everywhere else.

Reply:
Fixed in a couple of cases I could find